mlperf_logging/package_checker/package_checker.py

```diff
@@ -94,6 +127,7 @@ def check_training_result_files(folder, ruleset, quiet, werror):
                     _EXPECTED_RESULT_FILE_COUNTS[benchmark],
                     len(result_files),
                 ))
+                too_many_errors = True
```

Comment:
No, my intention here was set too_may_errors set whenever an error is detected -- let's discuss at the meeting because I don't really like how errors are handled in the package checker and for sure I don't want to make the situation worse than before.

Reply:
So it seems that yes, the test would pass if you submitted more runs. Adding this will make the checker more robust. However we need to have a good testing infra and error detection scheme for the checkers.